pkg/es/config/config.go

```diff
@@ -29,6 +29,7 @@ import (
 	"sync"
 	"time"
 
+	elasticsearch8 "github.com/elastic/go-elasticsearch/v8"
```

Comment:
URL is invalid

Reply:
Yes, it is probably because of the specific version that we have used here i.e. v8. github.com/elastic/go-elasticsearch is a valid url.